wordpress-mobile/WordPress-Android
# Add WP media source to media picker

Fixes #12640  

This PR introduces the WP media library as another source in the consolidated media picker. There were several obstacles which I'll try to describe here.
- Identifiers - the local media uses URI as an identifier. The media items in the media library used remote ID (and they don't have URIs). To get around this I've created a class called `Identifier` that has 2 implementations - `LocalUri` for local files and `RemoteId` for items in the media library. These identifiers replace the selected URIs and are used in the whole component. There is a subclass called Parcel used to save the selected identifiers into the saved instance state
- the `load` method now uses `loadMore` parameter instead of passing in page size and offset

What's missing:
- Figure out tracking

To test:
- Go to GB editor/Add image block/Click "Add media"
- Select "WordPress Media Library"
- Check the selector behaviour (multiselection is allowed)
- Select a few items
- Confirm selection
- Multiple Image blocks are added to the post (for each selected items)

To test 2:
- Check that the previous behaviour is not broken with the "Choose from device" option

PR submission checklist:
